veps/sig-compute/opt-out-rbac-aggregation.md

```diff
@@ -0,0 +1,230 @@
+# VEP 160: Opt-out of RBAC Role Aggregation
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [ ] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
+- [ ] (R) Target version is explicitly mentioned and approved
+- [ ] (R) Graduation criteria filled
+
+## Overview
+
+KubeVirt currently deploys ClusterRoles with `aggregate-to-*` labels that automatically aggregate KubeVirt-related permissions to the default Kubernetes roles (`admin`, `edit`, `view`). This means that by default, any user with these standard Kubernetes roles automatically receives corresponding KubeVirt permissions within their namespace.
+
+This VEP proposes adding a new configuration option in the KubeVirt CR that allows cluster administrators to opt-out of this default RBAC aggregation behavior. When disabled, KubeVirt ClusterRoles will not be aggregated to the default Kubernetes roles, requiring explicit RBAC assignments for KubeVirt resources.
+
+## Motivation
+
+While RBAC aggregation provides convenience for many deployments, security-conscious environments often require more granular control over permissions. The current behavior presents challenges for organizations that:
+
+- Need to comply with security policies requiring explicit permission grants rather than automatic inheritance
+- Want to implement the principle of least privilege where Kubernetes admin roles do not automatically include VM management capabilities
+- Require audit trails showing deliberate RBAC assignments for KubeVirt resources
+- Have multi-tenant environments where VM management permissions must be explicitly delegated
+
+By providing an opt-out mechanism, cluster administrators gain the flexibility to choose between:
+1. **Convenience** (default): Automatic aggregation where users with standard Kubernetes roles automatically get KubeVirt permissions
+2. **Control**: Explicit RBAC management where KubeVirt permissions must be manually assigned
+
+## Goals
+
+- Allow cluster administrators to disable RBAC role aggregation via a configuration option in the KubeVirt CR
```

Comment:
```suggestion
- Allow cluster administrators to disable RBAC role aggregation
```
Doing this in the CR is not a goal of itself, it is a byproduct of  the `dynamic toggling` goal.